spec/services/rapid_ready_for_decision/hypertension_pdf_generator_spec.rb

```diff
@@ -6,7 +6,9 @@
 require 'lighthouse/veterans_health/client'
 
 RSpec.describe RapidReadyForDecision::HypertensionPdfGenerator, :vcr do
-  subject { PDF::Inspector::Text.analyze(pdf_generator.generate.render).strings }
+  subject { PDF::Inspector::Text.analyze(generated_pdf.render).strings }
+
+  let(:generated_pdf) { pdf_generator.generate }
```

Comment:
Not sure of the desire for this change, but if you want to keep it, let's rename the variable so that the call to `render` makes more intuitive sense:
```suggestion
  subject { PDF::Inspector::Text.analyze(compiled_pdf.render).strings }

  let(:compiled_pdf) { pdf_generator.generate }
```
Another idea is perhaps the `generate` method is misnamed?